docs/api-reference.md

```diff
@@ -691,7 +691,7 @@ import {
 
 ### HTTP API
 
-Resolve provides a standard HTTP API that allows you to send aggregate commands and query Read and View Models. Refer to the [Standard HTTP API](curl.md) topic for more information.
+Resolve provides a standard HTTP API that allows you to send aggregate commands and query Read and View Models.
```

Comment:
Should "Resolve" be "reSolve"? (Just checking.)

Reply:
```suggestion
reSolve provides a standard HTTP API that allows you to send aggregate commands, and query Read and View Models.
```

Reply:
Yes, it should be reSolve, thank you